cmake/cpptoml/CMakeLists.txt

```diff
@@ -17,10 +17,13 @@ project(cpptoml-build CXX)
 
 include(ProcessorCount)
 ProcessorCount(N)
-if(NOT N EQUAL 0 AND ((${CMAKE_VERSION} VERSION_GREATER "3.12.0") OR ${CMAKE_VERSION} VERSION_EQUAL "3.12.0"))
-  set(CMAKE_BUILD_FLAGS -j${N})
-elseif(LINUX OR QNX)
-  set(CMAKE_BUILD_FLAGS -- -j${N})
+
+if(NOT N EQUAL 0)
+    if(((${CMAKE_VERSION} VERSION_GREATER "3.12.0") OR ${CMAKE_VERSION} VERSION_EQUAL "3.12.0"))
```

Comment:
I think there was a decision to drop ubuntu 16.04 support. I should create an issue to update `cmake_minimum_required` to `(VERSION 3.10)` then this could be simplified to `if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")`.

Which cmake version do we have on Debian 10, Raspbian, mac and Windows?